readme.md

```diff
@@ -43,12 +43,58 @@
 
 ## Test
 
-- `./eng/Generate.ps1` (at root in PowerShell Core)
-- `dotnet test` (at root)
+**`./eng/Generate.ps1` (at root in PowerShell Core)**
+
+This tests changes across many swagger definitions and samples.
```

Comment:
Was this supposed to be "This test ..." or "These tests change ..."?

Reply:
How about:
`This command tests your change across many swagger definitions and samples`